subsys/bluetooth/controller/Kconfig

```diff
@@ -576,4 +579,171 @@ config BT_CTLR_DEBUG_PINS
 
 endif # BT_LL_SW
 
+if BT_LL_KW41Z
```

Comment:
Organization-wise I think it would be better to separate all of these options into a separate Kconfig file.
controller/hci/vendor/<nxp_vendor_name>/Kconfig would be my preference.

Reply:
yes, I agree.